docs/streaming-programming-guide.md

```diff
@@ -1787,11 +1788,11 @@ reliable file system (e.g., HDFS, S3, etc.) to which the checkpoint information
 This is done by using `streamingContext.checkpoint(checkpointDirectory)`. This will allow you to
 use the aforementioned stateful transformations. Additionally,
 if you want to make the application recover from driver failures, you should rewrite your
-streaming application to have the following behavior.
+streaming application to have the following behavior:
```

Comment:
A couple of the trivial formatting changes I agree with; most of the changes are not improvements (e.g. "being" is slightly more accurate below). I would close this PR